scripts/.bash_robotics

```diff
@@ -0,0 +1,30 @@
+# Source ros workspace
+source /opt/ros/foxy/setup.bash
+
+# Source python virtual enviroment
+source ~/Programming/robotics-prototype/venv/bin/activate
```

Comment:
The following is missing here:
```
source ~/Programming/robotics-prototype/robot/rospackages/install/local_setup.bash
```

Additionally including these from the old setup script here would be nice:
```
ROBOTICS_WS="/home/$USER/Programming/robotics-prototype"
BASE="$ROBOTICS_WS/robot/basestation"
ROVER="$ROBOTICS_WS/robot/rover"
ROSPACKAGES="$ROBOTICS_WS/robot/rospackages"

alias ..="cd .."
alias b="cd -"
alias robotics="cd $ROBOTICS_WS"
alias base="cd $BASE"
alias rover="cd $ROVER"
alias arm="cd $ROVER/ArmDriverUnit"
alias wheels="cd $ROVER/MobilePlatform"
alias rostings="cd $ROSPACKAGES"
alias mcu="cd $ROSPACKAGES/src/mcu_control_python/mcu_control_python"
alias util="cd $ROBOTICS_WS/robot/util"'
```

Reply:
What is local_setup.bash for? Can it simply be added to the bash_robotics file?

For the aliases i thought it would be better for people to create their own.

Reply:
I see, my thought process was that these aliases are just nice to have in general and I tend to use them a lot. But I'll leave this up to you.

The local_setup.bash is actually for the ROS workspace. It's essentially the ROS2 equivalent of doing `source rospackages/devel/setup.bash`

Reply:
decided to add them cause wtv